pom.xml

```diff
@@ -320,6 +320,11 @@
       <artifactId>grizzled-slf4j_${scala.version.prefix}</artifactId>
       <version>1.0.3</version>
     </dependency>
+
+    <!--
+      Only used directly in tests, but adding test-scope blocks transitive inclusion via Spark, resulting in runtime
```

Comment:
My understanding was that something under `src/test` (and nothing under `src/main`) imported something from this dep, which is why we had it `test`-scoped, in our pom.

We were forced to widen the scope due to Maven excluding it from being packaged as a trans-dep from other deps of ours, like Spark.

After digging just now, I find that nothing in our code-base uses this dep, so I've removed it; I don't know why it was there originally.